server/server.py

```diff
@@ -192,14 +193,19 @@ async def details(request: Request):
 
 
 @app.get("/api/tts")
-async def tts(text: str, speaker_id: str, style_wav: str):
-    style_wav = style_wav_uri_to_dict(style_wav)
+async def tts(speaker_id: str, text: str = Query(min_length=1)):
+    if speaker_id not in new_speaker_ids.keys():
+        return JSONResponse(
+            status_code=406,
+            content={"message": f"Client did not request a supported media type.", "accept": list(new_speaker_ids.keys())},
```

Comment:
per security suggestion of fastapi it is better to raise an HTTPError than to return a json. https://fastapi.tiangolo.com/tutorial/handling-errors/#raise-an-httpexception-in-your-code